README.md

```
@@ -1,30 +1,105 @@
+| フレームワーク   | Vite + React 19                                         | Hono on Cloudflare Workers  |
+| UI/スタイル      | Tailwind CSS v4 + React Aria Components + Framer Motion | —                           |
+| 認証             | Clerk (`@clerk/clerk-react`)                            | Clerk (`@clerk/backend`)    |
+| DB               | —                                                       | Neon Postgres + Drizzle ORM |
+| ストレージ       | —                                                       | Cloudflare R2               |
+| ルーティング     | react-router-dom v7                                     | Hono Router                 |
+| テスト           | —                                                       | Vitest                      |
+| Linter/Formatter | oxlint + oxfmt                                          | oxlint + oxfmt              |
+
+## ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)